### ISSUE TITLE
[BUG] Issues with napari => 0.2.6

### ISSUE THREAD

**Issue by adamltyson** [SainsburyWellcomeCentre/amap-python#15]
Importing amap with newer versions of napari gives the following error:
```python
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-5-74ed835f7ff4> in <module>
      1 get_ipython().run_line_magic('gui', 'qt5')
      2 import napari
----> 3 from amap.vis.vis import prepare_load_nii

~/miniconda3/envs/pyclub/lib/python3.7/site-packages/amap/vis/vis.py in <module>
      3 from argparse import ArgumentParser, ArgumentDefaultsHelpFormatter
      4 from natsort import natsorted
----> 5 from napari.util.io import magic_imread
      6 
      7 from pathlib import Path

ModuleNotFoundError: No module named 'napari.util'
```

Need to update use of napari's API.

**Comment by adamltyson** [comment]
Just need to change `napari.util` to `napari.utils` (https://github.com/napari/napari/pull/808)

**Comment by adamltyson** [comment]
Fixed in 0a103db65a9ee8482e49ac1b24b9d8987c70477a
